plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/HorseStats.java

```diff
@@ -32,6 +33,9 @@ public void onHorseStatCheck(PlayerInteractEntityEvent event) {
 		if (!item.getType().equals(config.getHorseCheckerItem())) {
 			return;
 		}
+		if (event.getHand() == EquipmentSlot.OFF_HAND) {
```

Comment:
Maybe I'm just paranoid, but I'd defensively replace this with `if (event.getHand() != EquipmentSlot.HAND)` instead.